data/categories.txt

```diff
@@ -618,6 +618,7 @@ ja:コンビニエンスストア|コンビニ|買い物
 ko:편의점|쇼핑
 nl:Buurtwinkel|kruidenier|winkel
 nb:Nærbutikk|nærbutikk|butikk
+pl:Sklep ogólnospożywczy|sklep osiedlowy|sklep
```

Comment:
Yes, it's rather official and common name for a small shop where you could buy food, press, cigarettes and other stuff.